include/cadmium/engine/pdevs_dynamic_simulator.hpp

```diff
@@ -27,6 +27,8 @@
 #include <cadmium/modeling/dynamic_model.hpp>
 #include <cadmium/modeling/dynamic_message_bag.hpp>
 #include <cadmium/engine/pdevs_dynamic_engine.hpp>
+#include <cadmium/logger/dynamic_common_loggers.hpp>
+#include <cadmium/logger/common_loggers.hpp>
```

Comment:
You always need this one to make the dynamic logger work? Maybe it should be included in the other one.

Reply:
I don't use common_loggers.hpp in the dynamic_common_loggers.hpp file and I not need it to make the dynamic_common_loggers.hpp work, the common_logger is here because it is used by the pdevs_dynamic_simulator.hpp simulator class.